restful.module

```diff
@@ -339,7 +339,14 @@ function restful_delivery($var = NULL, $method = 'format') {
       ->getFormatterManager();
     $formatter_manager->setResource($resource);
     $plugin_definition = $resource->getPluginDefinition();
-    $formatter_name = isset($plugin_definition['formatter']) ? $plugin_definition['formatter'] : NULL;
+    if ($request->getMethod() == RequestInterface::METHOD_OPTIONS) {
```

Comment:
Because formatters can do complex things based on other metada of the field. Like, the field normally is an Entity Reference, but when turned into an OPTIONS call it does not reference any entity. Moving that into relationships and trying to find an ID for the referenced entity doesn't make a lot of sense.